tmt/trying.py

```diff
@@ -680,28 +720,29 @@ def go(self) -> None:
             self.action_verbose(plan)
 
         # Choose the initial action
-        action = Action.START_ASK
+        action = find_action("start_ask")
         if self.opt("login"):
-            action = Action.START_LOGIN
+            action = find_action("start_login")
         elif self.opt("ask"):
-            pass  # already START_ASK
+            pass  # already start_ask
         elif self.tests:
-            action = Action.START_TEST
+            action = find_action("start_test")
 
         # Loop over the actions
         try:
             while True:
                 # Choose the verbose and debug level
-                if action in [Action.VERBOSE, Action.DEBUG]:
-                    getattr(self, f"prompt_{action.action}")()
+                action_name = action.full_name
+                if action_name in ["verbose", "debug"]:
+                    getattr(self, f"prompt_{action_name}")()
 
                 # Handle the individual actions
                 for plan in self.plans:
                     plan.header()
-                    getattr(self, f"action_{action.action}")(plan)
+                    action.func(self, plan)
 
                 # Finish for keep and quit
-                if action in [Action.KEEP, Action.QUIT]:
+                if action_name in ["keep", "quit"]:
```

Comment:
Add an attribute to `ActionInfo`, "when invoked, break the loop". Then we can drop this list of actions.

Reply:
added `exit_loop` attribute